agent/doc/agent-doc.md

```diff
@@ -19,5 +19,6 @@ The first service check will start right now, the second one at 12:00:06, third
 We don't care about the duration of tests, we work with time points. 
 In the previous example, the second check for the first service will be scheduled at 12:00:10 even if all other checks has not been yet started.
 
-In case of check duration is too long, we might exceed maximum of concurrent checks. In that case checks will be executed as soon one will be ended.
+In case of check duration is too long, we might exceed maximum of concurrent checks. In that case checks will be executed as soon one will be ended. But next calls will occur at check start + check_period.
```

Comment:
I don't understand the new sentence.

Reply:
I suppose the new sentence explain what you describe in the new example below.

Reply:
deleted